### PR TITLE
[EDU-3647] Update Templates reference and Guides page

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue:

[EDU-3647]

### Changes

Update Templates reference and Guides page to include CTA/Links to new templates (wave2)

### Additional links

n/a.

[EDU-3647]: https://aziontech.atlassian.net/browse/EDU-3647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx [hunk 1]
```diff
@@ -214,6 +238,98 @@ The QStash EdgeFunction Scheduler template sets up and manages a custom edge fun
...
+
+</Button>
+<br />
+<Button href="https://manager.azion.com/build-application/build/choose-template/" text="deploy this template" variant="secondary"> 
+
+</Button>
+
+#### Fauna Boilerplate
+
+This boilerplate  allows you to deploy a database directly on the **Azion Edge Platform**. The template includes the creation of a static interface to facilitate you to create, retrieve, update, and delete items in your collections.
```
Comment (gabriel-azion) [inline src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx:323]:
```suggestion
This boilerplate allows you to deploy a database directly on the **Azion Edge Platform**. The template includes the creation of a static interface to facilitate you to create, retrieve, update, and delete items in your collections.
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx [hunk 2]
```diff
@@ -140,6 +140,30 @@ The Azion Vue Boilerplate provides an automation solution to build a Vue Single-
 
 </Button>
 
+#### Hexo Boilerplate
+
+Hexo Boilerplate facilitates you to build and run a Hexo Single-Page Application (SPA) directly on the edge by automating several steps, from repository management to edge deployment.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx:145]:
```suggestion
Hexo Boilerplate enables you to build and run a Hexo Single-Page Application (SPA) directly on the edge by automating several steps, from repository management to edge deployment.
```

File: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx [hunk 3]
```diff
@@ -214,6 +238,98 @@ The QStash EdgeFunction Scheduler template sets up and manages a custom edge fun
...
+
+</Button>
+<br />
+<Button href="https://manager.azion.com/build-application/build/choose-template/" text="deploy this template" variant="secondary"> 
+
+</Button>
+
+#### Hello World
+
+The Azion Hello World template deploys an edge application that prints a Hello, World! message in your browser and you can use it as a starting point to become familiar with Azion Edge Platform and edge computing capabilities.
```
Comment (bru-andrade) [inline src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx:259]:
```suggestion
The Azion Hello World template deploys an edge application that prints a Hello, World! message in your browser and you can use it as a starting point to become familiar with the Azion Edge Platform and its edge computing capabilities.
```

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx [hunk 4]
```diff
@@ -213,6 +237,97 @@ O template QStash EdgeFunction Scheduler configura e gerencia uma edge function
...
+
+</Button>
+<br />
+<Button href="https://manager.azion.com/build-application/build/choose-template/" text="implante o template" variant="secondary"> 
+
+</Button>
+
+#### Hello World
+
+O template Hello World da Azion cria uma edge application que exibe uma mensagem Hello, World! no seu navegador e você pode usá-la como ponto de partida para se familiarizar com a Plataforma de Edge da Azion e os recursos de edge computing.
```
Comment (bru-andrade) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:258]:
```suggestion
O template Hello World da Azion cria uma edge application que exibe uma mensagem Hello, World! no seu navegador. Você pode usá-la como ponto de partida para se familiarizar com a Plataforma de Edge da Azion e os recursos de edge computing.
```

**Review by bru-andrade — APPROVED** [review]
👯‍♀️

File: src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx [hunk 5]
```diff
@@ -213,6 +237,97 @@ O template QStash EdgeFunction Scheduler configura e gerencia uma edge function
...
+
+</Button>
+<br />
+<Button href="https://manager.azion.com/build-application/build/choose-template/" text="implante o template" variant="secondary"> 
+
+</Button>
+
+### Integração com banco de dados
+
+Implante uma aplicação integrada com um banco de dados para ser executada diretamente no edge da Azion e aproveite os recursos que oferece edge computing.
```
Comment (hannah-sk) [inline src/content/docs/pt-br/pages/menu-principal/recursos-adicionais/templates-e-integracoes/templates-visao-geral.mdx:304]:
```suggestion
Implante uma aplicação integrada com um banco de dados para ser executada diretamente no edge da Azion e aproveite os recursos oferecidos pelo edge computing.
```

**Review by hannah-sk — APPROVED** [review]

**Pull request merged**